### PR TITLE
New step - tslint

### PR DESCRIPTION
**Pull request by ibratoev**
A new step for running TSLint. It automatically installs TSLint if not available.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/tslint/0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+source_code_url: https://github.com/ibratoev/steps-tslint
+support_url: https://github.com/ibratoev/steps-tslint/issues
+published_at: 2017-03-22T16:03:54.333538005+02:00
+source:
+  git: https://github.com/ibratoev/steps-tslint.git
+  commit: 764d22569364b38e572a1f0c8783e26beca79aee
+host_os_tags:
+- osx
+- ubuntu
+project_type_tags:
```
Comment (godrei) [inline steps/tslint/0.0.2/step.yml]: following project types are available on bitrise:

- ios
- android
- xamarin
- react-native
- cordova
- ionic

is this step available only for ios and android?

File: steps/tslint/0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- ios
+- android
+type_tags:
```
Comment (godrei) [inline steps/tslint/0.0.2/step.yml]: following type_tags are available on bitrise:

- access control
- artifact info
- installer
- deploy
- utility
- dependency
- code sign
- build
- test
- notification

could you please select one (or more) from the list?

_if you think non item of the list fits to this step, please let me know_

**Comment by godrei** [comment]
Hi @ibratoev ,

thanks for the step!

Please take a look on my comments and please create a new PR with only one commit. To do this please:

- close this PR
- remove the `tslint-0.0.2` branch from your steplib fork
- share the step again

**Comment by viktorbenei** [comment]
@ibratoev can you please re-share the step as advised?

I'll close this PR but if you have any questions just let us know!

**Comment by viktorbenei** [comment]
Just one comment: I'd suggest you to create a new version with the fixes, instead of re-sharing the 0.0.2 version. Simply fix + create 0.0.3 + share that.

**Comment by ibratoev** [comment]
Thank you @viktorbenei. I will fix it and send 0.0.3 for review in the next couple of days.

**Comment by viktorbenei** [comment]
Awesome, thank you @ibratoev and let us know if we can help with anything!

**Pull request closed without merging**